src/clr-addons/searchfield/search-field.scss

```diff
@@ -0,0 +1,28 @@
+@include exports('searchfield.clr-addons') {
+  .search-input-wrapper {
+    position: relative;
+
+    .search-input {
+      padding-left: 1rem;
```

Comment:
please use a padding-left of 1.2rem, as the text now sticks a bit too much to the glass.
and also a padding-right of 1rem is needed, otherwise the text overflows with the remove-symbol